census/controllers/admin.js

```diff
@@ -66,10 +69,31 @@ let loadQuestionSets = function(req, res) {
     });
 };
 
+let buildIndexSite = function(req, res) {
+  /*
+  Generate and deploy an Index site for the site specified in the request.
+  */
+
+  // Set env for process to include DEBUG option, for more comprehensive
+  // logging.
+  const env = process.env;
+  // env.DEBUG = 'metalsmith-godi-*';
+
+  exec(`node index/generate.js ${res.locals.domain} -dsy ${res.locals.surveyYear}`,
```

Comment:
`res.locals` has been set somewhere on the server so it's trusted data to inject into exec?